docs/headers.md

```diff
@@ -229,17 +229,17 @@ See [F.23: Use a `not_null<T>` to indicate that “null” is not a valid value]
 ##### Construct/Copy
 
 ```cpp
-template <typename U, typename = std::enable_if_t<std::is_convertible<U, T>::value>>
+template <typename U, std::enable_if_t<std::is_convertible<U, T>::value, bool> = true>
 constexpr not_null(U&& u);
 
-template <typename = std::enable_if_t<!std::is_same<std::nullptr_t, T>::value>>
+template <std::enable_if_t<!std::is_same<std::nullptr_t, T>::value, bool> = true>
 constexpr not_null(T u);
 ```
 
 Constructs a `gsl_owner<T>` from a pointer that is convertible to `T` or that is a `T`. It [`Expects`](#user-content-H-assert-expects) that the provided pointer is not `== nullptr`.
 
 ```cpp
-template <typename U, typename = std::enable_if_t<std::is_convertible<U, T>::value>>
+template <typename U, std::enable_if_t<std::is_convertible<U, T>::value, bool> = true>
```

Comment:
I can't figure out how to bypass any of these `enable_if_t`s. (Though I have little confidence my attempts were exhaustive).

Could you provide an example of why this change is necessary?

Reply:
I'll try to come up with one example per change.